src/capture.cpp

```diff
@@ -1055,6 +1053,8 @@ void acquisition::Capture::run_soft_trig() {
     catch(...){
         ROS_FATAL_STREAM("Some unknown exception occured. \v Exiting gracefully, \n  possible reason could be Camera Disconnection...");
     }
+    ros::shutdown();
```

Comment:
@PushyamiKaveti in current dev branch ros::shutdown() is being called in class deconstructor.
could you rebase and submit pr, so we can merge this into dev.
thanks